### PR TITLE
Add inheritance topic docs

### LINKED ISSUE #3036: task: Docs - Clarify what's inherited in the schema (e.g. icon inheritance)
### Component

Documentation & Examples

### Task Description

From @ubaumann on discord:
> The icon is not inherited (like the label). I assume this is on purpose; is it somewhere documented? It is more the point that I did not find the in the documentation if it is inherited or not. I tried it to set it on the Generic Interface and it was not passed to the inherited instance. So I added it for each interface. I think that is totally fine if it intended. Would maybe be nice to know what is not inherited from a Model

### PR DESCRIPTION
**Pull request by BeArchiTek**
Fixes #3036

### CODE REVIEW

File: docs/docs/topics/schema.mdx [hunk 1]
```diff
@@ -197,120 +282,127 @@ nodes:
...
 ```
 
-:::info
+### Inherited properties
 
-If a `human_friendly_id` is defined on a node but no `uniqueness_constraints` are present, a uniqueness constraints matching the `human_friendly_id` will be automatically created.
+#### List of inheritable properties
 
-If at least one attribute is marked as unique, and `human_friendly_id` hasn't been defined, `human_friendly_id` will be automatically created.
+- **`human_friendly_id`**: A user-friendly identifier for the entity.
```
Comment (wvandeun) [inline docs/docs/topics/schema.mdx]: I would not make them bold and in a code block. It looks a bit weird in the rendered docs to me.
Not a blocking thing though

**Review by wvandeun — APPROVED** [review]
Minor comment, but LGTM

File: docs/docs/topics/schema.mdx [hunk 2]
```diff
@@ -197,120 +282,127 @@ nodes:
...
+  - name: Device
+    namespace: Infra
+    inherit_from:
+      - "GenericDevice"
+    attributes:
+      - name: role
+        kind: Dropdown
+        optional: true
+        choices:
+          - name: core
```
Comment (dgarros) [inline docs/docs/topics/schema.mdx:327]: I think we could drop 2 or 3 choices from this list to keep it short and easier to read

**Review by dgarros — APPROVED** [review]
Looks great, thanks

**Pull request merged**